node/bin/src/config.rs

```diff
@@ -423,9 +423,13 @@ pub struct ProverApiConfig {
     /// however, we won't turn real FRI proofs into fake ones - even on timeout.
     pub fake_snark_provers: FakeSnarkProversConfig,
 
-    /// Timeout after which a prover job is assigned to another Fri Prover Worker.
+    /// Timeout after which a FRI prover job is assigned to another Fri Prover Worker.
+    #[config(alias = "job_timeout", default_t = Duration::from_secs(300))]
+    pub fri_job_timeout: Duration,
```

Comment:
Should we make this number more sensible? I assume 60s is about enough, WDYT?

Reply:
similar time